### DISCUSSION TITLE
Tags remain empty on Search Result Cards

### DISCUSSION THREAD

**Discussion by Limerick-gh** [microsoft-search/pnp-modern-search#3995] [Q&A] [answered]
Hi there
I am just starting with PnP Modern Search and really appreciate this great enhancement for SharePoint search.

Setting everything up, I am struggling to get the Tags displayed properly on Card Layout.
Only empty gray boxes show up:
![image](https://github.com/user-attachments/assets/1ac632aa-404f-4a9a-94af-6888a353ab6a)

This is the configuration:
LayoutSlots:
![image](https://github.com/user-attachments/assets/44f7dd59-2eac-4130-8649-f42fe1a02802)

And this is visible when having Debug enabled:
![image](https://github.com/user-attachments/assets/7d4e4739-a53c-4244-922c-b93ef683a1d9)

The content is coming from a MultiChoice column that I mapped to the Managed Property.

Any help is appreciated to have the Tags displayed.

It's probably just something simple?

Thanks in advance for taking a look into it.

**Comment by kasperbolarsen** [comment, marked as answer]
Tags are usually managed Metadata, aka terms from the termstore. I guess it should be possible to tweak the handlebars expression to handle a multi value choice felt, but I haven't tested it

**Reply by Limerick-gh** [reply]
Hi @kasperbolarsen 
Thanks for pointing to the right direction.
I switched my tags from Multiple Choice to a Managed Metadata field and had to enable localization. Otherwise the shown values were garbled in the search filter, even I used the same refinablestring and it was ok in the search results. Anyhow, with some CSS tweaks I am quite happy now.
![image](https://github.com/user-attachments/assets/beb99cda-3c74-43e8-a373-8b973e891aff)


Again thanks for your hint, that helped!
